roles/setup_repo/tasks/PG_Debian_setuprepos.yml

```diff
@@ -29,6 +29,15 @@
     state: present
   become: true
 
+# Exclusive to pgd setup authentication
```

Comment:
This actually isn't exclusive to pgd.  It affects everything as long as we allow the default pg_hba.conf file to be created with ident auth.  So we want this for everyone.